squad/frontend/tests.py

```diff
@@ -117,7 +117,6 @@ def get(cls, build, page, search, per_page=50):
             id__in=table.__get_page_filter__(page, per_page)
         ).prefetch_related(
             Prefetch('test_run', queryset=TestRun.objects.only('environment')),
-            'suite',
```

Comment:
can you ellaborate why `suite` is being removed? is it not used for anything?

Reply:
If you are already traversing a model, that model will be automatically prefetched. Ex:
```python
.prefetch_related('suite__metadata')
```
because there's a prefetch of `suite__metadata`, `suite` is automatically prefetched, so no need to explicitly prefetch it